src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -1054,7 +1054,7 @@ Namespace Microsoft.CodeAnalysis.Operations
             Dim condition As Lazy(Of IOperation) = New Lazy(Of IOperation)(Function() Create(boundDoLoopStatement.ConditionOpt))
             Dim body As Lazy(Of IOperation) = New Lazy(Of IOperation)(Function() Create(boundDoLoopStatement.Body))
             Dim ignoredConditionOpt As Lazy(Of IOperation) = New Lazy(Of IOperation)(Function()
-                                                                                         If doLoopKind = DoLoopKind.None Then
+                                                                                         If doLoopKind = doLoopKind.None Then
```

Comment:
`DoLoopKind.None`

Reply:
Fixed. I think this is the VS merge tool behaving weirdly.

---
In reply to: [148031164](https://github.com/dotnet/roslyn/pull/22842#discussion_r148031164) [](ancestors = 148031164)